### ISSUE TITLE
Parsing error: invalid-first-character-of-tag-name.

### ISSUE THREAD

**Issue by jasonxia23** [vuejs/eslint-plugin-vue#474]
**Tell us about your environment**

* **ESLint Version:** 3.19.0
* **eslint-plugin-vue Version:** 
* **Node Version:** 8.9.0

**Please show your full configuration:**

I am using `standard` library.

**What did you do? Please include the actual source code causing the issue.**

```vue
<div class="value">
  <pre class="element">{{ `<${item.element.tagName} ${item.element.attributes.map(attr => attr.key + '="' + attr.value + '"').join(' ')} />` }}</pre>
</div>
```

**What did you expect to happen?**

No need to throw eslint error.

**What actually happened? Please include the actual, raw output from ESLint.**

![image](https://user-images.githubusercontent.com/15098719/39904905-190747c4-550d-11e8-94d3-34e57201d21b.png)

**Comment by armano2** [comment]
@jasonxia23 you should consider using `:is="item.element.tagName"` but i this string should not be parsed anwyay

**Comment by jasonxia23** [comment]
I'm not using `item.element.tagName` as a tagName but a template string.

**Comment by michalsnik** [comment]
It looks like an upstream issue in `vue-eslint-parser`

**Comment by jasonxia23** [comment]
@mysticatea Thanks for your answer.
Still quite weird for me, especially the issue you referenced.
And I thought everything in `pre` should be displayed as it was put in. If you put `&lt;` in a `pre` tag then `&lt;` will be displayed. Obviously Vue has transpiled the entities.

**Comment by mysticatea** [comment]
Nope. `<pre>` elements keep whitespaces (includes line breaks). But tags and entities are translated as same as outside of `<pre>` elements.

The special elements which keep tags in that content are only `<title>`, `<textarea>`, `<script>`, and `<style>`.

**Comment by jasonxia23** [comment]
Thanks for figure it out
